examples/minimum_blending.py

```diff
@@ -0,0 +1,41 @@
+"""
+Minimum blending
+================
+
+Demonstrates how to use the `minimum` blending mode with inverted colormaps.
+`minimum` blending uses the minimum value of each R, G, B channel for each pixel.
+`minimum` blending can be used to yield multichannel color images on a white 
+background, when the channels have inverted colormaps assigned.
+An inverted colormap is one where white [1, 1, 1] is used to represent the lowest 
+values, as opposed to the more conventional black [0, 0, 0].
```

Comment:
@andy-sweet Should I add mention of the new ones added to the GUI?

Reply:
Yes, I think just giving one example of an inverted colormap name (maybe one you use below) would be sufficient.